.pre-commit-config.yaml

```diff
@@ -8,12 +8,31 @@ repos:
       - id: mixed-line-ending
       - id: requirements-txt-fixer
       - id: trailing-whitespace
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: v1.5
+    hooks:
+      - id: autopep8
   - repo: https://gitlab.com/pycqa/flake8
     rev: 3.7.9
     hooks:
       - id: flake8
+        additional_dependencies:
+          - flake8-blind-except
+          - flake8-builtins
+          - flake8-docstrings
+          - flake8-import-order
+          - flake8-rst-docstrings
+          - pep8-naming
   - repo: https://github.com/kynan/nbstripout
     rev: 0.3.7
     hooks:
       - id: nbstripout
         files: ".ipynb"
```

Comment:
you still have to change this to local repos

Reply:
Ah ok, so like pylint? The reason why pylint is local is that it does runtime checks as well.

Reply:
yeah I thought that was the idea, that things are coupled now right?